scaffolder-templates/advanced-workflow/template.yaml

```diff
@@ -107,62 +108,146 @@ spec:
                 persistencePSQLUserKey:
                   title: PostgreSQL User key from secret
                   type: string
-                  description: The key name in which the PostgreSQL user is stored.
+                  description: The key name in which the PostgreSQL user is stored
                   default: postgres-username
                 persistencePSQLPasswordKey:
                   title: PostgreSQL Password key from secret
                   type: string
-                  description: The key name in which the PostgreSQL password is stored.
+                  description: The key name in which the PostgreSQL password is stored
                   default: postgres-password
                 persistencePSQLServiceName:
                   title: PostgreSQL K8s Service Name
                   type: string
                   default: sonataflow-psql-postgresql
-                  description: Name of the service running the PostgreSQL instance.
+                  description: Name of the service running the PostgreSQL instance
                 persistencePSQLServiceNamespace:
                   title: PostgreSQL K8s Service Namespace
                   type: string
                   default: sonataflow-infra
-                  description: Namespace of the service running the PostgreSQL instance.
+                  description: Namespace of the service running the PostgreSQL instance
                 persistencePSQLServicePort:
                   title: PostgreSQL Port
                   type: integer
                   default: 5432
-                  description: Port on which the PostgreSQL instance is running.
+                  description: Port on which the PostgreSQL instance is running
                 persistencePSQLDatabaseName:
                   title: PostgreSQL Database Name
                   type: string
-                  description: Name of the database to use for persistence.
+                  description: Name of the database to use for persistence
                   default: sonataflow
               required:
                 - namespace
                 - argocdNamespace
-                - quayOrgName
-                - quayRepoName
                 - persistencePSQLSecretName
                 - persistencePSQLUserKey
                 - persistencePSQLPasswordKey
                 - persistencePSQLServiceName
                 - persistencePSQLServiceNamespace
                 - persistencePSQLServicePort
                 - persistencePSQLDatabaseName
+    - title: Quay Configuration
+      required:
+        - quayConfig
+      properties:
+        quayConfig:
+          title: Create a new Quay repository with the provided name?
+          type: string
+          default: "Use Existing Repository"
+          enum:
+            - "Use Existing Repository"
+            - "Create New Repository"
+            - "No Quay Config Needed"
```

Comment:
If the CI/CD was set to `None` this page should not be displayed to the user, is it possible to add an `if` statement to skip this page?

Otherwise, we should display an empty page with a constant string saying "No CI/CD selected, ignoring quay" or something similar